signingscript/exceptions.py

```diff
@@ -1,23 +1,45 @@
+"""Signingscript exceptions."""
 from scriptworker.exceptions import ScriptWorkerTaskException
 from scriptworker.constants import STATUSES
 
 
 class TaskVerificationError(ScriptWorkerTaskException):
+    """Something went wrong during task verification."""
+
     def __init__(self, msg):
+        """Initialize TaskVerificationError.
```

Comment:
To be honest, I don't see value to the docstrings added in this file 😕 I guess it's here to make flake8_docstring happy. If that's the case, can we configure it to be less strict? 

For example, I believe this docstring doesn't add to much because:

* It doesn't add more arguments to a regular `Exception`.
* It doesn't provide more information about how or when to use this exception
* Unlike scriptworker, signingscript is not meant to be used by a dependent project. Hence, people who will work with this exception, will likely be working on Signingscript itself. Hence, they can take a look at the implementation.